File: examples/Delphi/Compound/ModelEvolution/Version1/dModel.pas
unit dModel;

interface

uses
  Classes, 
  Controls, 
  Forms, 
  BoldSubscription, 
  BoldModel, 
  BoldHandle, 
  BoldUMLModelLink,
  BoldUMLRose98Link, BoldAbstractModel, BoldUMLMMLink;

type
  TdmModel = class(TDataModule)
    Model: TBoldModel;
    BoldUMLMMLink1: TBoldUMLMMLink;
    BoldUMLRoseLink1: TBoldUMLRoseLink;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  dmModel: TdmModel;

implementation

{$R *.DFM}

end.

File: examples/CBuilder/Compound/ModelEvolution/Version2/dModel.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "dModel.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "BoldAbstractModel"
#pragma link "BoldHandle"
#pragma link "BoldModel"
#pragma link "BoldSubscription"
#pragma link "BoldUMLModelLink"
#pragma link "BoldUMLRose98Link"
#pragma resource "*.dfm"
TdmModel *dmModel;
//---------------------------------------------------------------------------
__fastcall TdmModel::TdmModel(TComponent* Owner)
  : TDataModule(Owner)
{
}
//---------------------------------------------------------------------------
